lib/tests.nix

```diff
@@ -101,7 +101,8 @@ let
           # config for tests to make them run faster or work at all
           documentation.enable = false;
           hardware.enableAllFirmware = lib.mkForce false;
-          boot.initrd.preDeviceCommands = ''
+          # FIXME: we don't have an systemd in stage-1 equialvent for this
+          boot.initrd.preDeviceCommands = lib.mkIf (!config.boot.initrd.systemd.enable) ''
```

Comment:
This is an unrelated drive-by, right? It doesn't seem to be related to the rest of the PR, and the test still built fine multiple times without it. What's it for? Something to do with the discussion in #651?